src/libraries/System.Formats.Cbor/src/System.Formats.Cbor.csproj

```diff
@@ -3,6 +3,7 @@
     <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppPrevious);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPackable>true</IsPackable>
+    <UseIntellisensePackageDocXmlFile>false</UseIntellisensePackageDocXmlFile>
```

Comment:
Need to remove this property enabling the feature in the 3 assemblies, for now.

Reply:
Cbor doesn't use `IsPartialFacadeAssembly`, nor does it use `GeneratePlatformNotSupportedAssemblyMessage` so that property can stay here and we can use the project's produced intellisense file.

Reply:
Ah right, it was the only assembly that passed the test results.

Reply:
Alrught, I will enable it for the Brotli assembly but in a follow-up PR because it is currently failing due to some public APIs not being documented. I don't want to address all those errors here.